### PR TITLE
Set local_config default value to true

### PR DESCRIPTION
**Pull request by laura-ding**
The documentation no longer provides configuration-related command documentation, so the default is to use the configuration file configuration.

### CODE REVIEW

File: conf/nebula-graphd.conf.production [hunk 1]
```diff
@@ -29,6 +31,8 @@
 # Whether to treat partial success as an error.
 # This flag is only used for Read-only access, and Modify access always treats partial success as an error.
 --accept_partial_success=false
+# The high watermark ratio of system memory
+--system_memory_high_watermark_ratio=0.8
```
Comment (yixinglu) [inline conf/nebula-graphd.conf.production:35]: this flag has existed in this conf

File: conf/nebula-graphd.conf.default [hunk 2]
```diff
@@ -31,6 +33,8 @@
 # Whether to treat partial success as an error.
 # This flag is only used for Read-only access, and Modify access always treats partial success as an error.
 --accept_partial_success=false
+# The high watermark ratio of system memory
+--system_memory_high_watermark_ratio=0.8
```
Comment (yixinglu) [inline conf/nebula-graphd.conf.default:37]: ditto

File: conf/nebula-graphd.conf.production [hunk 3]
```diff
@@ -7,6 +7,8 @@
 --enable_optimizer=true
 # Heartbeat interval of communication between meta client and graphd service
 --heartbeat_interval_secs=10
+# Whether to use the configuration obtained from the configuration file
+--local_config=true
```
Comment (CPWstatic) [inline conf/nebula-graphd.conf.production:11]: Should the default value be  false?

Comment (laura-ding) [reply conf/nebula-graphd.conf.production:11]: The documentation no longer provides configuration-related command documentation, so the default is to use the configuration file configuration.

**Review by yixinglu — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**